next.config.js

```diff
@@ -3,6 +3,7 @@ module.exports = withSass({
     cssModules: true,
     publicRuntimeConfig: {
         API_URL: process.env.API_URL,
+        GUID_URL: (process.env.NODE_ENV==='staging'||'development')?(`https://testswitch-candidate-staging.herokuapp.com`):(`https://testswitch-candidate.herokuapp.com`),
```

Comment:
This does work, but instead lets just do `GUID_URL: provess.env.CANDIDATE_SITE_URL`, and then add an env var.
That way its super easy to configure - and you can make it work locally too if you like.